src/Classes/MyRadio/URLUtils.php

```diff
@@ -82,6 +82,16 @@ public static function redirectWithMessage($module, $action, $message)
     }
 
     /**
+     * Redirects to another page, specified already by the caller.
+     *
+     * @param string  $URI The relative URI to redirect to.
+     */
+    public static function redirectURI($URI)
+    {
+        header('Location: '.$URI);
```

Comment:
Is it worth restricting what this can redirect to (e.g. to ury.org.uk) in case of something evil? Probably not...